content/components/action-menu.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Action menu
+react: https://primer.style/react/ActionMenu
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=17091%3A66640
+description: Action menu is a component based on Action list for creating a menu of actions that expands through a trigger button.
+---
+
+import {Box, Button, Heading, Link} from '@primer/react'
+
+## Usage
+
+### In overlays
```

Comment:
Aren't action menus _always_ in an overlay?

Reply:
I think so Mike, yes 🤔 it just _is_ an overlay menu

Reply:
So maybe we don't need the "In overlays" heading?